src/AutoEnrollPlugin/Service/ValidationService.cs

```diff
@@ -0,0 +1,243 @@
+using System;
+using System.Linq;
+using Serilog;
+
+using Innovatrics.SmartFace.Integrations.AutoEnrollPlugin.Models;
+
+namespace Innovatrics.SmartFace.Integrations.AutoEnrollPlugin.Services
+{
+    public class ValidationService : IValidationService
+    {
+        private readonly ILogger logger;
+
+        private static readonly Func<(Notification notification, StreamMapping streamMapping), bool> validateFaceQuality = (input) =>
```

Comment:
code style: 
What lead you to prefer `private static readonly Func<()> ....`  over
`private static bool ValidateFaceQuality(Notification notification, StreamMapping streamMapping)`

I would highly prefer the latter